docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,325 @@
-## Questions
+## Questions and Quizzes
 
-**Question component consists of a question body, a hint and an answer.**
+Question and quiz components provide an easy way to test readers on the relevant content topic in the page.
 
-<include src="codeAndOutput.md" boilerplate >
-<variable name="highlightStyle">html</variable>
-<variable name="code">
-<question>
-  The question body. <md>:grey_question:</md>
-  <div slot="hint">
-    Question hint. <md>:crystal_ball:</md>
-  </div>
-  <div slot="answer">
-    Question answer. <md>:pencil:</md>
-  </div>
+#### Introduction
+
+Question components (`<question>`) can be one of the following types: **Mcq**, **Checkbox** or **Text**.
+
+<div class="px-5">
+<question type="checkbox" header="Which of the following is true? (**Card Header**)" hint="This is the question hint, you can use **markdown** here as well.">
+
+Anything you place inside a question not in a slot is inserted into the **card body**!
+
+<q-option reason="We aren't concatenating strings!">
+  20 + 20 = 2020
+</q-option>
+<q-option reason="Division by zero is **undefined**!">
+  1 / 0 = infinity
+</q-option>
+<q-option correct>
+11 / 11 = 1
+</q-option>
 </question>
-</variable>
-</include>
+</div>
+
+
+In all cases, note that content directly inserted in between `<question>...</question>` will be inserted into the card's body.
 
+To output your markup to the header, or insert content into the hint tip box, simply insert your markup inside the `header` and `hint` attributes respectively:
```

Comment:
`To output your markup to the header`: rephrase in a non-programmer friendly way?